deploy/infra/main.bicep

```diff
@@ -29,14 +32,46 @@ module db './webapi/db.bicep' = {
   }
 }
 
-module webApi './webapi/webapp.bicep' = {
-  name: 'webAppDeploy'
+module dbArc './webapi/dbArc.bicep' = if (customLocationId != '') {
+  name: 'dbArcDeploy'
   params: {
     location: location
+    customLocationId: customLocationId
+    dataControllerId: dataControllerId
+    subscriptionId: subscription().id
+    resourceGroupName: resourceGroup().name
+    namespace: 'appservice-ns' //Should be derived from customLocation
+    dbName: substring(uniqueString(dbName),0,5) //Name length restrictions
+    administratorLoginPassword: postgresAdminPassword
+  }
+}
+
+module webApiAzure './webapi/webappAzure.bicep' = if (customLocationId == '') {
+  name: 'webAppAzureDeploy'
+  params: {
+    location: location
+    workspaceId: monitoring.outputs.workspaceId
+    appSettingsPgHost: customLocationId == '' ? dbAzure.outputs.pgHost : dbArc.outputs.pgHost //The ternary operator is needed to ensure deployment dependencies are correct
+    appSettingsPgUser: customLocationId == '' ? dbAzure.outputs.pgUser : dbArc.outputs.pgUser //The ternary operator is needed to ensure deployment dependencies are correct
+    appSettingsPgDb: customLocationId == '' ? dbAzure.outputs.pgDb : dbArc.outputs.pgDb //The ternary operator is needed to ensure deployment dependencies are correct
+    appSettingsNodeEnv: webapiNodeEnv
+    appSettingsPgPassword: postgresAdminPassword
```

Comment:
spell out postgres in the name?

Reply:
I see a combo of pg and postgres. Maybe standardize on one?  It's a p3

Reply:
There is lavck of consistensy with the eventual env var being used in code. That one, by convention for postgres uses PGHOST, PGDB etc. Too many cascading changes to make this in this pr.